index.html

```diff
@@ -31,9 +31,144 @@ <h1 class="introduction"> Hey there. I’m Chukwuemeka <br> <span> I’m a softw
+                                <ul>
+                                        <li><a href="#">Ruby on rails</a></li>
+                                        <li><a href="#">css</a></li>
+                                        <li><a href="#">Javascript</a></li>
+                                        <li><a href="#">html</a></li>
+                                </ul>        
+                                <button class="resumebutton" type="submit"> See Project</button>        
+        
+                            </div>
+                </div>  
+            </div>
+            
+ <!-- About me Section            -->
+
+            <article id="aboutme">
+                    <h2> About me</h2>
+                    <p class="describeme"> I can help you build a product ,<br> feature or website Look through <br>
+                         some of my work and experience! If <br> you like what you see and have a<br>
+                          project you need coded, <br>
+                            don’t hestiate to contact me.</p>
+                            <button class="resumebutton" type="submit">Get My Resume</button>
+            </article>
+            <div class="grid-container"> 
+                    <div class="aboutmebox">
+                        <img src="img/languages.png" alt="language details">
+                        <p class="describeaboutmebox">Languages</p>
+                        <ul>
+                                <li><a href="#">Javascript</a></li>
+                                <li><a href="#">Ruby</a></li>
+                                <li><a href="#">Html</a></li>
+                                <li><a href="#">CSS</a></li>
+                        </ul>  
+                    </div>
+                    <div class="aboutmebox">
+                            <img src="img/frameworks.png" alt="frameworks details">
+                            <p class="describeaboutmebox">Frameworks</p>
+                            <ul>
+                                    <li><a href="#">Bootstrap</a></li>
+                                    <li><a href="#">Ruby on rails</a></li>
+                                    <li><a href="#">Rspec</a></li>
+                                    <li><a href="#">Capybara</a></li>
+                                    <li><a href="#">Selenium</a></li>
+                            </ul> 
+                    </div>
+                    <div class="aboutmebox">
+                            <img src="img/skills.png" alt="skills details">
+                            <p class="describeaboutmebox">Skills</p>
+                            <ul>
+                                    <li><a href="#">Codekit</a></li>
+                                    <li><a href="#">Github</a></li>
+                                    <li><a href="#">Codepen</a></li>
+                                    <li><a href="#">Gitlab</a></li>
+                                    <li><a href="#">Terminal</a></li>
```

Comment:
- [ ] I think it's better to use the Figma template family font as required by the previous reviewer. Creates a better UI.